cypress/e2e/signIn.cy.js

```diff
@@ -29,6 +29,11 @@ describe('Sign In page', () => {
   });
 
   it('should not provide an ability to log in with wrong credentials', () => {
-
+    signInPage.visit();
+    signInPage.typeEmail(user.email);
+    signInPage.typePassword(user.password);
```

Comment:
Using the correct password in a test for wrong credentials might not be appropriate. Ensure that the password used here is incorrect to properly test the failure scenario.